omod/src/main/webapp/fragments/field/concept.gsp

```diff
@@ -0,0 +1,41 @@
+<%
+	import groovy.json.*
+    config.require("formFieldName")
+    config.require("conceptId")
+    
+    def maxResults = config.maxResults ? config.maxResults : 20;
+    def conceptAnswers = new JsonBuilder(conceptAnswers);
+%>
+
+<script type="text/javascript">
+    jq(function() {
+    var consAnswers = ${conceptAnswers};
```

Comment:
I would also expect this concept field to be more generic, you should not assume that it will only be used for coded concepts, someone might to have a drop down of concepts of a given class or even any concept.